hw/ip/socdbg_ctrl/data/socdbg_ctrl.hjson

```diff
@@ -0,0 +1,345 @@
+               '''
+    },
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_creator_seed_sw_rw_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Provision enable qualifier coming from life cycle controller.
+               This signal enables SW read / write access to the RMA_TOKEN and CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_seed_hw_rd_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Seed read enable coming from life cycle controller.
+               This signal enables HW read access to the CREATOR_ROOT_KEY_SHARE0 and CREATOR_ROOT_KEY_SHARE1.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_dft_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Test enable qualifier coming from life cycle controller.
+               This signals enables the TL-UL access port to the proprietary OTP IP.
+               '''
+    }
+    { struct:  "lc_tx"
+      type:    "uni"
+      name:    "lc_check_byp_en"
+      act:     "rcv"
+      default: "lc_ctrl_pkg::Off"
+      package: "lc_ctrl_pkg"
+      desc:    '''
+               Life cycle partition check bypass signal.
+               This signal causes the life cycle partition to bypass consistency checks during life cycle state transitions in order to prevent spurious consistency check failures.
+               '''
+    }
+  ]
+
+  // REGISTER definition
+  regwidth: "32"
+  registers: {
+    core: [
+        // CTRL register
+        { name: "DEBUG_POLICY_CTRL"
+          desc: "Debug Policy Control"
+          swaccess: "rw"
+          hwaccess: "hro"
+          fields: [
+            { bits: "3:0"
+              resval: "0"
+              name: "debug_policy_level"
+              desc: '''
+                     Debug Policy Control Setting
+                     Indicates the current debug authorization policy that is
+                     distributed to the rest of the SoC to govern debug / DFT
+                     feature unlock
+                    '''
+            }
+          ]
+        }
+        { name: "DEBUG_POLICY_VALID"
+          desc: "Debug Policy Valid"
+          swaccess: "rw"
+          hwaccess: "hro"
+          fields: [
+            { bits: "0"
+              resval: "0"
+              name: "debug_policy_valid"
+              desc: '''
+                     Debug Policy Control Setting
+                     Indicates the current debug authorization policy that is
+                     distributed to the rest of the SoC to govern debug / DFT
+                     feature unlock
+                    '''
+            }
+          ]
+        }
+        { name:     "STATUS_MBX"
```

Comment:
QQ: the actual authorization exchange is going to be handled via the JTAG mailbox, is that right?

Reply:
Yes; The auth exchange will be via the JTAG mailbox. The results will be communicated via the socdbgctrl module